surface/src/3rdparty/opennurbs/opennurbs_archive.cpp

```diff
@@ -15318,15 +15318,24 @@ const wchar_t* ON_FileIterator::NextFile()
   for(;;)
   {
     current_file_attributes = 0;
+    /*
+      from readdir man page:
+      If the end of the directory stream is reached,
+      NULL is returned and errno is not changed. If an error occurs,
+      NULL is returned and errno is set appropriately.
+    */
+    errno = 0;
```

Comment:
It seems to me that setting errno to zero and comparing it to zero afterwards is not even necessary, since we don't need to distinguish between end of stream and error? In both cases, `break;` follows, so checking the returned value (`dp`) should be sufficient, right?

Reply:
Yes, you are right. I havel deleted errno checking.